internal/pkg/aws/ssm/ssm.go

```diff
@@ -132,3 +133,16 @@ func convertTags(inTags map[string]string) []*ssm.Tag {
 	}
 	return tags
 }
+
+// GetSecretValue retrieves the value of a secret parameter from AWS Systems Manager Parameter Store.
```

Comment:
nit:

Because an SSM parameter is not always a "secret" (in SSM's terminology). An SSM param could be a `PlainString`, which is not encrypted; it could be a `SecretString`, which _is_ encrypted (and can be thought of as a "secret" in SSM's terminology).



```suggestion
// GetSecretValue retrieves the value of a parameter from AWS Systems Manager Parameter Store.
```